src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { IonicPageModule } from 'ionic-angular';
+import { CpcEndTestModal } from './cpc-end-test-modal';
+import { ModalResultItemComponent } from './components/modal-result-item/modal-result-item';
+
+@NgModule({
+  declarations: [
+    CpcEndTestModal,
```

Comment:
should we look at upper casing the category in same format as pages

Reply:
changed